tedge/src/cli/certificate/cli.rs

```diff
@@ -56,11 +56,15 @@ impl BuildCommand for TEdgeCertCli {
 
             TEdgeCertCli::Upload(cmd) => {
```

Comment:
Presumably it's not just the upload that you would need this for, but also the connect.

Reply:
BTW - ignoring the server certificate validity - is that part of this story?

Reply:
> Presumably it's not just the upload that you would need this for, but also the connect.

The issue exhibits in upload as upload is using HTTP GET and POST when trying to upload upload device certificate.

> BTW - ignoring the server certificate validity - is that part of this story?

It's not in AC, but if not implemented it still will be unusable due to invalid certificate.

Reply:
Not unusable, because the customer can upload a different self-signed certificate to the edge VM. But even if they do that, they will still need a good error to show them they need to add that as a trusted root.

The broken generated certificate is being fixed in edge.